Server/BackEndMonitoringServer/include/CContainerOfLogicalDisk.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "CHardwareStatusSpecification.h"
+
+class CLogicalDiskInfo;
+
+class CContainerOfLogicalDisk : public CHardwareStatusSpecification
```

Comment:
Is CContainerOfLogicalDisk a CHardwareStatusSpecification? Looks like the better option will be CContainerOfLogicalDisk has a CHardwareStatusSpecification. So use composition or private inheritance.
Try to use the same logic in all places where inheritance is used.

Reply:
Done for CContainerOfLogicalDisk and CContainerOfProcesses